### ISSUE TITLE
isMounted(...) is deprecated in plain JavaScript React classes

### ISSUE THREAD

**Issue by slonoed** [QubitProducts/react-test-tree#19]
```
Warning: isMounted(...) is deprecated in plain JavaScript React classes. Instead, make sure to clean up subscriptions and pending requests in componentWillUnmount to prevent memory leaks.
```

https://github.com/QubitProducts/react-test-tree/blob/master/lib/testTree.js#L44

**Comment by oliverwoodings** [comment]
@slonoed thanks for this, I believe I should be able to replace the call internally with React.findDOMNode, will do some testing.

**Comment by oliverwoodings** [comment]
Actually just using React.findDOMNode on it's own won't work since it throws exceptions if the node isn't found. Easiest option will probably be to wrap findDOMNode in a try catch, although that does feel a tad hacky... simpler than the other solution, however, which would be to wrap componentDidMount and componentWillUnmount and then maintain our own internal mounted state tracker.

**Comment by oliverwoodings** [comment]
Fixed via #20
